### ISSUE TITLE
Not detected: `StudioDisplay/0.5`

### ISSUE THREAD

**Issue by Moonbase59** [matomo-org/device-detector#7456]
[AzuraCast](https://github.com/AzuraCast/AzuraCast) uses your library and [pointed me here](https://github.com/AzuraCast/AzuraCast/issues/6556).

I was wondering why my `StudioDisplay` is reported as `Unknown browser 0.00`, instead of (at least) `StudioDisplay 0.5`.

[StudioDisplay](https://github.com/Moonbase59/studiodisplay) is a modular MQTT-based signalling & display solution for Web Radio Stations. The software is around since 2017.

It has a [stream metadata reader module](https://github.com/Moonbase59/studiodisplay/blob/master/docs/install-raspberry-pi.md#live-radio-stream-title-display) that reads in-stream metadata from Icecast and Shoutcast audio streams and publishes that to an MQTT server.

Its user agent looks like this (examples):
```
StudioDisplay/0.5 (Linux; 5.15.0-79-generic; x86_64; 64bit; latin1)
StudioDisplay/0.5 (Windows; 10; AMD64; 64bit; utf-8)
StudioDisplay/0.5 (Darwin; 21.4.0; x86_64; 64bit; utf-8)
StudioDisplay/0.5 (Linux; 5.10.63-v7+; armv7l; 32bit; utf-8)
```
where
- `StudioDisplay` - name (doesn’t change)
- `0.5` - version*
- `Linux` - system**
- `5.10.63-v7+` - release**
- `armv7l` - machine type**
- `32bit` - bitness**
- `utf-8` - encoding expected for in-stream metadata***

\* = This is the software version. Known versions so far are `0.4` and `0.5`.  
\*\* = It uses a Python3 module, so these depend on what the `platform` module reports.
\*\*\* = typically `utf-8` or `latin1` but can be empty (=guess) or have other values.

It is difficult to categorize, since it’s no browser but concentrates on detecting if an audio stream is live, and reads ICY metadata for displaying purposes ("now playing"). It is also not a bot or a crawler but typically watches _this one stream defined in its configuration_.

Maybe it could be categorized as a (non-mobile) _app_, or, better yet, a _media player_, since it behaves exactly like a media player: Connects to an audio stream and reads it (although throwing away all audio information and just keeping the in-stream metadata).

I’d still be happy if it was at least detected as `StudioDisplay 0.5` and not "unknown". It _is_ of interest to station operators (like AzuraCast) to see which listeners are actually listeners and which are just dumb displays hanging at studio walls.

Together with other data, this is typically used by web radio studios for a wall-mounted studio display like this:

![grafik](https://github.com/matomo-org/device-detector/assets/3706922/45e08611-bffe-4e34-ac55-2df3a37ffaf6)

**Comment by sgiehl** [comment]
Should be fixed with #7457

**Comment by Moonbase59** [comment]
Thanks, appreciated!
